firefox-ios/Client/Frontend/Browser/BrowserViewController/Actions/GeneralBrowserAction.swift

```diff
@@ -65,6 +65,7 @@ enum GeneralBrowserActionType: ActionType {
     case showPasswordGenerator
     case didSelectedTabChangeToHomepage
     case enteredZeroSearchScreen
+    case didUnhideToolbar
```

Comment:
open to putting this under another ToobarAction or another action, but the action seems more general than toolbar since it not an action taken on the toolbar itself but surrounding circumstances